Trimble.Modus.Components/Converters/DescriptionToRowHeightConverter.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Globalization;
+
+namespace Trimble.Modus.Components.Converters
+{
+    public class DescriptionToRowHeightConverter : IValueConverter
```

Comment:
Can you explain what this does and why this is needed?

Reply:
If the description is present for the text cell the height is 56 else its 54

Reply:
It doesn't work for the template cell. Try to define the height in the TextCell class

Reply:
Done